### PR TITLE
[rac_yidgha] Turn help file into php header format

### PR DESCRIPTION
**Pull request by LornaSIL**
Help file was not in correct format

### CODE REVIEW

File: release/rac/rac_yidgha/source/help/rac_yidgha.php [hunk 1]
```diff
@@ -97,6 +93,4 @@
 <li><a href='http://keymankeyboards.com/iso:ydg'>Other keyboards for these languages</a></li>
```
Comment (darcywong00) [inline release/rac/rac_yidgha/source/help/rac_yidgha.php]: Maybe this link instead?
https://keyman.com/keyboards/languages/ydg

**Comment by rachitralli** [comment]
@DavidLRowe @MakaraSok

ى 0649
 06CCی

These above two characters have the same appearance but the correct is 06CCی
 please change the same in all layouts. I have already requested to change the same. Thanks

**Comment by LornaSIL** [comment]
@rachitralli This PR is closed. You should either create a new issue or you should create a new PR making the change you are desiring.

**Comment by rachitralli** [comment]
@LornaSIL thanks you very much, the PR creation button has been disabled in my computer. I am unable to create Issue or PR in my computer, I don't know why? please enable the PR creation button for me so that I can create PR and issues etc. thanks

**Pull request merged**